tests/test_export.py

```diff
@@ -254,57 +247,3 @@ def test_export_with_category_path_failure(self):
                 exporter.export()
 
 
-class SanitizeHtmlTests(unittest.TestCase):
```

Comment:
It absolutely makes sense to remove these tests, should we have a test for `Post.htmlize()`? It will just be mocking `sanitize_html` and making sure it is called with the right arguments. If you think that's overkill though I understand.